test/unit/WebsocketServer.test.js

```diff
@@ -151,6 +167,26 @@ describe('WebsocketServer', () => {
             })
         })
 
+        it('adds the subscription id to signed messages', (done) => {
```

Comment:
This and the other new tests look like unnecessary test duplication! `WebsocketServer` is probably mostly agnostic to whether the message is signed or unsigned, isn't it? What lines of code in `WebsocketServer.test.js` are you testing with this test case, which aren't covered by the other tests?

If you look at the diff for `WebsocketServer.js` in this PR, actually only a few lines have changed from the previous version: `publisher.publish(...)` is being called with some new arguments for publish requests with signatures. **That** is the test case you should be adding!